### ISSUE TITLE
adios mascota

### ISSUE THREAD

**Issue by problema** [RoDNet/Bugtracker-Infinium-4.0.6a#856]
Bueno el problema es simple y sencillo. De buenas a primeras mi mascota desapareció y no me dejaba llamarla aunque en "llamar mascota" salía. al relogear y cerrar wow, ya no me salía ni eso, es decir me quedé sin mascota hasta que capturé otra. he mirado en el establo y no está. Por lo que el tigre del comienzo de los elfos de la noche lo he perdido. Tengo constancia de otros pjs a los que le ha pasado lo mismo, con distintas mascotas.

Un gm me ha dado como solución borrar el pj y volver a crearlo, pero ¿Para qué hacerlo si sé de otras personas que le ha pasado lo mismo? ¿ Quién me dice que crearme uno nuevo hará que no me vuelva a desaparecer? 

Espero que lo arregléis pronto. 
![WoWScrnShot_031713_211436](https://f.cloud.github.com/assets/3883161/268567/8c155a42-8f3f-11e2-961a-590de08abae5.jpg)

**Comment by Dekadencee** [comment]
No, no borres tu pj, eso no arreglará tu error, es un error global.
Estamos aplicando un fix temporal en el cual relogueando, recuperarás tu mascota, paciencia.

**Comment by problema** [comment]
ok! muchas gracias! esperemos que funcione!

**Comment by adrianrp1988** [comment]
Es un bug causado por los slots al parecer, creo que una posible solucion seria manualmente poniendole una pet en la db al slot 0 para que aparezca, no estoy seguro

**Comment by Walkum** [comment]
El bug es un lamentable sistema de pets que tiene el emulador, lo que pasa es que a los dueños del emulador se les ha reportado y desde hace más de 3 meses dicen "hay que reescribirlo entero" pero no lo hacen, cuando el sistema realmente sí va bien y el único bug es este, que se puede solucionar pero no quieren.

Toca esperar a ver si podemos solucionarlo nosotros.

**Comment by problema** [comment]
por lo que me comenta por la hermandad, otro problema sobre el mismo tema es que los huecos en el establo seguirán ocupados por las mascotas desaparecidas....
